docs/pages/desktop-access/getting-started.mdx

```diff
@@ -27,104 +25,123 @@ This guide requires you to have:
 
-From the start menu of your Domain Controller, search for “Active Directory Users and Computers” and open the corresponding program.
+```powershell
+$Name="Teleport Service Account"
+$SamAccountName="svc-teleport"
+New-ADUser `
+  -Name $Name `
+  -SamAccountName $SamAccountName `
+  -AccountPassword (Read-Host "Set Password (and note it down)" -AsSecureString) `
+  -Enabled $true
+```
 
-Find your domain in the side menu, and right click the container you want the service account to live in (typically Users). From the context
-menu, select `New > User` and fill out the fields for your Teleport LDAP Service Account. For the "User logon name" field, you can use `svc-teleport`
-On the password screen, unselect  “User must change password at next login” and select “Password never expires”. Make a note of what values you put 
-for user logon name and password, which will be used in your `teleport.yaml` configuration file later, and then click through the rest of the wizard.
+Next, in the same PowerShell prompt, enter the following commands to limit your service account's TODO permissions to the bare minimum necessary for
+Teleport to operate.
 
-<Admonition
-  type="note"
-  title="Default User Permissions"
->
-  By default, domain users have Read and List permissions throughout Active Directory. If you have configured
-  any restrictions to Domain Users, you may have to adjust which groups this account is a member of.
-</Admonition>
+```powershell
+# Save your domain's distinguished name to a variable.
+$DomainDN=$((Get-ADDomain).DistinguishedName)
 
-<Figure align="left" bordered caption="Create Service Account">
-  ![Create Service Account](../../img/desktop-access/ad-new-user.png)
-</Figure>
+# Create the CDP/Teleport container.
+# If the command fails with "New-ADObject : An attempt was made to add an object to the directory with a name that is already in use",
+# it means the object already exists and you can move on to the next step.
+New-ADObject -Name "Teleport" -Type "container" -Path "CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN"
 
-Now right click the same container you did before, and select `New > Group`. Choose a Group name like `svc-teleport-group` and ensure that “Global”
-is selected for “Group scope”, and “Security” for “Group type”, then hit "OK". Make note of the Group name you chose, as it will
-be used later for the PowerShell variable `$TeleportLDAPGroupName`.
+# Gives Teleport the ability to create LDAP containers in the CDP container.
+dsacls "CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CC;container;"
+# Gives Teleport the ability to create and delete cRLDistributionPoint objects in the CDP/Teleport container.
+dsacls "CN=TELEPORT,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CCDC;cRLDistributionPoint;"
```

Comment:
Not sure if these are case sensitive or not, but the code uses `Teleport`, not `TELEPORT`, so I would make it match just to be safe.